lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -48,6 +48,14 @@ class OpenResponseMetadataSerializer(serializers.Serializer):
     name = serializers.CharField(source='display_name')
     prompts = serializers.ListField()
     type = serializers.SerializerMethodField()
+    textResponseConfig = serializers.SerializerMethodField()
+    fileUploadResponseConfig = serializers.SerializerMethodField()
+
+    def get_textResponseConfig(self, instance):
+        return instance.text_response if instance.text_response else 'none'
+
+    def get_fileUploadResponseConfig(self, instance):
+        return instance.file_upload_response if instance.file_upload_response else 'none'
```

Comment:
A few notes:
1. Don't return 'none'. Return a null/none type, e.g. None.
2. A better shorthand for this `return a.field if a.field else default` in Python is `return a.get('field', default)`
3. But you probably wouldn't want to do that. There's [Serializer field setup](https://www.django-rest-framework.org/api-guide/fields) that lets you specify that in the config.